cmd/util/cmd/execution-state-extract/execution_state_extract_test.go

```diff
@@ -139,13 +140,12 @@ func TestExtractExecutionState(t *testing.T) {
 				// generate random block and map it to state commitment
 				blockID := unittest.IdentifierFixture()
 
+				lctx := lockManager.NewContext()
+				require.NoError(t, lctx.AcquireLock(storage.LockInsertOwnReceipt))
 				require.NoError(t, storageDB.WithReaderBatchWriter(func(rw storage.ReaderBatchWriter) error {
-					lockManager := storage.NewTestingLockManager()
-					lctx := lockManager.NewContext()
```

Comment:
Moving NewContext outside of WithReaderBatchWriter, so that it's only released after the batch is committed.